api_types.ts

```diff
@@ -576,7 +575,7 @@ export interface CommonPackFormulaDef<T extends ParamDefs> {
    * },
    * ```
    */
-  validateParameters?: MetadataFormulaDef<ExecutionContext, boolean>;
+  validateParameters?: MetadataFormula;
```

Comment:
Can this be parameterized with `boolean`?

Reply:
Needed to make MetadataFormula support a second generic parameter. I added that with any as the default and applied it here but leaving remaining usage untouched